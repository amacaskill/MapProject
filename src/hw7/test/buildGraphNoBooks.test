# builds graph without books

LoadGraph g buildGraphNoEdges2.tsv

ListNodes g 

ListChildren g char1
ListChildren g char2
ListChildren g char3