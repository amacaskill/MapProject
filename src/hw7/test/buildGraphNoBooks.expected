# builds graph without books

loaded graph g

g contains: char1 char2 char3

the children of char1 in g are:
the children of char2 in g are:
the children of char3 in g are: